lib/config/default.js

```diff
@@ -519,12 +519,13 @@ defaultConfig.definition = () => ({
       env: 'NEW_RELIC_TRACER_ENABLED'
     },
     /**
-     * The duration at below which the slow transaction tracer should collect a
-     * transaction trace. If set to 'apdex_f', the threshold will be set to
+     * Sets the time, in seconds, for a transaction to be considered slow.
```

Comment:
i would remove the marking of slow transaction. it means that i'll collect a transaction trace

Reply:
I don't understand. How about an edit suggestion?